models/header.py

```diff
@@ -0,0 +1,49 @@
+def add_header(file_name: str, parameter: dict):
+    """
+    create a header introducing the values associated with each variable used in the file
+
+    Parameters:
+    ---------
+    file: str
+        name of the file without the header
+    parameters: dict
+        dictionary pairing each parameter with its default value
+
+    """
+    # name the new file
+    new_file_name = file_name.removesuffix(".bioMod") + "_with_variables.bioMod"
+
+    biomod_file = open(new_file_name, 'w')
+
+    # copy the first line of the original file onto the new file
+    biomod_file = open(new_file_name, 'w')
```

Comment:
is it necessary to do it twice? if not delete one of the lines.

Reply:
Done